plugins/help.lua

```diff
@@ -113,7 +113,37 @@ local action = function(msg, blocks, ln)
             api.sendKeyboard(msg.from.id, message, keyboard, true)
             return
         end
+        --NEW CODE
+        if is_mod(msg) then
+            keyboard = {
+                {
+                    { text = '🔰 User commands', callback_data = '!user' },
+                    { text = '🔰 Admin commands', callback_data = '!mod' },
+                },
+                {
```

Comment:
Instead of adding another keyboard rather take the if statement out of the currently working one so that its less likely errors will appear as there is currently not list here. If I understand that thats what you want to do, in which case I will do that quickly.